frontend/packages/console-app/src/components/modals/restore-pvc/restore-pvc-modal.tsx

```diff
@@ -191,7 +190,27 @@ const RestorePVCModal = withHandlePromise<RestorePVCModalProps>(
               loaded={pvcResourceLoaded}
               loadError={pvcResourceLoadError}
               pvcResource={pvcResource}
+              availableAccessModes={volumeSnapshotAnnotations?.[
+                snapshotPVCAccessModeAnnotation
+              ]?.split(',')}
             />
+            <FormGroup
+              fieldId="pvc-volume-mode"
+              className="co-restore-pvc-modal__input"
+              label="Volume Mode"
```

Comment:
Shouldn't this be traslated as well?

Reply:
yeah forgot to add that. Now added. thanks